mesh_segmenter/package.xml

```diff
@@ -6,8 +6,9 @@
   <maintainer email="alex.goins@swri.org">alex</maintainer>
   <license>Proprietary</license>
 
-  <depend>catkin</depend>
+  <buildtool_depend>catkin</buildtool_depend>
   <depend>vtk_viewer</depend>
 
+  <test_depend>rosunit</test_depend>
```

Comment:
I don't think we are actually using rosunit. We are just using gtest (or will be as soon as #22 gets merged)

Reply:
I removed it.